mitmproxy/contentviews/graphql.py

```diff
@@ -0,0 +1,60 @@
+import json
+
+import typing
+
+from mitmproxy.contentviews import base
+from mitmproxy.contentviews.json import parse_json
+
+
+PARSE_ERROR = object()
```

Comment:
This should be imported and not redefined here, or am I missing something?

Reply:
Wasn't sure :-) imported

Reply:
The idea here is to have a dedicated object representing "null"/error ([a sentinel object](https://python-patterns.guide/python/sentinel-object/#sentinel-objects)). We can't use `None` as `"null"` is valid JSON but also evaluates to `None`. And we don't want to raise because then it's not cached :) 
Long story short, to identify this it needs to be the same unique object. 😄